Jellyfin.Server.Implementations/Users/UserManager.cs

```diff
@@ -110,9 +127,14 @@ public class UserManager : IUserManager
                 throw new ArgumentException("Invalid username", nameof(name));
             }
 
-            // This can't use an overload with StringComparer because that would cause the query to
-            // have to be evaluated client-side.
-            return _dbProvider.CreateContext().Users.FirstOrDefault(u => string.Equals(u.Username, name));
+            using var dbContext = _dbProvider.CreateContext();
+
+            return dbContext.Users.Include(user => user.Permissions)
+                .Include(user => user.Preferences)
+                .Include(user => user.AccessSchedules)
+                .Include(user => user.ProfileImage)
+                .ToList()
```

Comment:
Why `ToList` if you're going to do `FirstOrDefault`?

Reply:
`ToList` brings the query to c# space instead of db space

Reply:
Forces the query to be evaluated client-side, since db's don't support the overload of `string.Equals` with a `StringComparison`

Reply:
That's sad :/

Reply:
I don't know if there's any real difference but there's also `.AsEnumerable()`.